etl/steps/data/garden/animal_welfare/2023-08-16/number_of_wild_fish_killed_for_food.meta.yml

```diff
@@ -0,0 +1,38 @@
+definitions:
+  common: |-
+    This data is an estimates of the number of fish caught in global fishing each year, using fisheries capture tonnages (from the Food and Agriculture Organization of the United Nations) for 2007-2016 and estimated mean weights for fish species.
+
+    This estimate does not include unrecorded fish capture, such as fish caught illegally and those caught as bycatch and discarded.
+tables:
+  number_of_wild_fish_killed_for_food:
+    variables:
+      n_wild_fish_low:
+        title: Estimated number of wild-caught fish (lower bound)
+        unit: 'fish'
+        short_unit: ''
+        description: |
+          Lower bound estimate of the number of wild-caught fish killed for food.
+
+          {definitions.common}
+        display:
+          numDecimalPlaces: 0
+      n_wild_fish:
+        title: Estimated number of wild-caught fish
```

Comment:
I am not sure we should calculate this variable - I think the data producers have purposely left a lot of uncertainty in the data because of the difficulty in estimating such a variable. I think producing a chart or map with this variable would give a false sense of accuracy.

Reply:
True, I'm also unsure. But I created it for consistency with other similar datasets where we also have lower and upper limits.